lib/trino-hdfs/src/main/java/io/trino/hdfs/HdfsEnvironment.java

```diff
@@ -58,7 +59,12 @@ public HdfsEnvironment(
 
     public Configuration getConfiguration(HdfsContext context, Path path)
     {
-        return hdfsConfiguration.getConfiguration(context, path.toUri());
+        return getConfiguration(context, path.toUri());
+    }
+
+    public Configuration getConfiguration(HdfsContext context, URI uri)
```

Comment:
Let's not add this method. If we're using this class, then we're still dependent on Hadoop and should continue to use `Path`.